api/dbConfig/seedDev.js

```diff
@@ -0,0 +1,6 @@
+const db = require('./init');
+const fs = require('fs');
+
+const seeds = fs.readFileSync(__dirname + '/dev_seeds.sql').toString();
+
+db.query(seeds, () => console.log('Dev database seeded'));
```

Comment:
Im making this comment because we did the same, but you do not need the dbConfig file and everything inside it to add things to the SQL table, you can add it from the same location. Essentially you made it way harder on yourselves.